docker-compose.live-reload.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+services:
+  sep-ui-live-dev:
+    build:
+      context: .
+      dockerfile: ./Dockerfile.dev
+    ports:
+      - '3000:3000'
+    volumes:
+      - .:/app
+    environment:
+      - PORT=3000
```

Comment:
Perhaps set this on the run command in Dockerfile, so the app works as expected even when not running through docker-compose.

Reply:
Hi Yatharth,

I have moved the environment variable settings to the Dockerfile, but I think we still can’t start it directly from the Dockerfile. It also needs to reference the host directory, and I am just copying the package.json into the Dockerfile. Everything else is used in bind mounts.